lib/client.js

```diff
@@ -277,7 +277,7 @@ exports.createClient = function(reference, name) {
       var retryRequest = function() {
         // Send request
         var sendRequest = function() {
-          debug("Calling: %s, retry: %s", entry.name, attempts - 1);
+          debug("Calling: %s %s, retry: %s", entry.method.toUpperCase(), entry.name, attempts - 1);
```

Comment:
http verb is implied from `entry.method`, so if you want this I propose:
`debug("Calling: %s %s (retry: 5s)", entry.method, url, attempts -1);`